### PR TITLE
Update MathJax version

### PR DESCRIPTION
**Pull request by donno2048**
This change will make it impossible to use the XSS from #192

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -43,7 +43,7 @@
       });
     </script>
 
-    <script async type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS_CHTML-full"></script>
+    <script async type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/MathJax.js?config=TeX-AMS_CHTML-full"></script>
```
Comment (oscarbenjamin) [inline templates/base.html:46]: Can you explain how this fixes it? Is this a known vulnerability in 2.7.0?

Comment (donno2048) [reply templates/base.html:46]: Yes, it was fixed in version 2.7.4

Comment (oscarbenjamin) [reply templates/base.html:46]: Do you have a link to anything describing the issue or explaining that it was fixed?

Comment (donno2048) [reply templates/base.html:46]: Sure,
https://groups.google.com/g/mathjax-dev/c/JCa8uRe3U2o

**Comment by asmeurer** [comment]
Thanks. We should actually update to mathjax 3 at some point as well.

I'm not sure if this will auto-deploy (I can't remember if we had it set up for Live). If it doesn't deploy let me know and I will do it manually.

**Comment by donno2048** [comment]
> I'm not sure if this will auto-deploy (I can't remember if we had it set up for Live). If it doesn't deploy let me know and I will do it manually.

The website is still using the old version...

**Comment by asmeurer** [comment]
OK, you have to make a release for the auto-deploy to work. I just did that.

**Comment by donno2048** [comment]
> OK, you have to make a release for the auto-deploy to work. I just did that.


Weirdly I still see that:

```sh
$ curl -s https://live.sympy.org/ | grep MathJax.js
    <script async type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS_CHTML-full"></script>
```

**Comment by asmeurer** [comment]
The deploy failed https://travis-ci.org/github/sympy/sympy-live/builds/774721074. I think it might be an authentication issue, but I'm not sure. @aktech do you have any thoughts?

**Comment by aktech** [comment]
Yeah, potentially or maybe gcloud version issue. I'll take a look.

**Comment by donno2048** [comment]
> > OK, you have to make a release for the auto-deploy to work. I just did that.
> 
> Weirdly I still see that:
> 
> ```shell
> $ curl -s https://live.sympy.org/ | grep MathJax.js
>     <script async type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS_CHTML-full"></script>
> ```

Seems like the two last builds failed

**Pull request merged**